velox/exec/prefixsort/tests/PrefixEncoderTest.cpp

```diff
@@ -350,18 +356,37 @@ TEST_F(PrefixEncoderTest, encodeHugeInt) {
   auto size = PrefixSortEncoderFactory::encodedSize(*type1);
```

Comment:
It takes a lot of effort to understand the intention here.
Please add some necessary comments to explain how ut ensures that all cases are covered（What scenarios are data patterns used to cover?）, especially the special paths introduced from spark & ​​duckDB mentioned above.

Reply:
I add the new decimal encodedSize, so we need to test for decimal.
Other logic for `PrefixSortEncoder` should be tested before.

Reply:
So currently we haven't figured out whether the two code branches mentioned above are covered by ut?

Reply:
It has covered. I add the tests to test decimal can cast to int and hugeint itself

Reply:
I think ut should be added here separately for them

Reply:
https://github.com/facebookincubator/velox/pull/10385/files#diff-669e68649d93c35ad69bd0724e11e6b181021954e5483076ad88604ee67d5fcbR291
hugeIntEncoder1_ tests cast to int case, and hugeIntEncoder2_ tests for hugeint

Reply:
hugeIntEncoder1_  https://github.com/facebookincubator/velox/pull/10385/files#diff-669e68649d93c35ad69bd0724e11e6b181021954e5483076ad88604ee67d5fcbR359 
hugeIntEncoder2_  https://github.com/facebookincubator/velox/pull/10385/files#diff-669e68649d93c35ad69bd0724e11e6b181021954e5483076ad88604ee67d5fcbR389